framework/util/page_guard_manager.cpp

```diff
@@ -102,6 +102,12 @@ static LONG WINAPI PageGuardExceptionHandler(PEXCEPTION_POINTERS exception_point
 #include <signal.h>
 #include <unistd.h>
 
+#ifdef __APPLE__
```

Comment:
Can these be made constants a la `const` or `constexpr` or such?

Reply:
Now `constexpr`